README.md

```diff
@@ -1,38 +1,85 @@
+# PCAP Analyzer
 
-This repo contains our pod's 2020 STEP Capstone project.
+PCAP Analyzer is the culmination of the 12 week [STEP internship program](https://buildyourfuture.withgoogle.com/programs/step/) at Google.
 
+
+## Table of Contents
+
+* [About the Project](#about-the-project)
+  * [Background](#background)
+  * [Team](#team)
+  * [Built With](#built-with)
+* [Setup](#setup)
+  * [Adding PCAP files](#adding-pcap-files)
+* [Benchmarks](#benchmarks)
+* [Contributing](#contributing)
+* [Demo](#demo)
+* [License](#license)
+
+## About The Project
+### Objective
+While most people use the internet in some kind of way, not everyone understands the basic fundamentals of how information can be transferred from one machine to another. The goal of this project is to provide causal users with a simple and intuitive way of visualising network communications. By explaining how computers communicate through packets and networks, we hope to educate users on the structure of the internet. 
+
+### Background
+While advanced solutions exist for high-level packet analysis, no current service focuses on casual users. Even to computer science students, the internet contains many interconnected components and protocols which can make it difficult to understand. Since 65% of people are visual learners, we hope to provide infographics and helpful depictions of the users’ input data. 
+
+### Overview
+Our goal is to help casual users understand and visualize where their computer packets are going and where they are coming from. We hope to have our website be both informative and interactive. 
+
+On the technical front, this means that to build out the PCAP analysis feature of our website, we will need to be able to do the following with our users’ files:
+
+1. File Upload / Data Collection ― since our main source of data are PCAP files, we can either enable PCAP file uploads or retrieve a PCAP file from a user-inputted URL.
+
+2. File Parsing / Data Processing ― we want to be able to extract key attributes from uploaded packets, including packet source, destination, protocol, and time. 
+
+3. File Storage / Data Retrieval ― in order to retrieve the above attributes, we need a way to store our packet information with easy access.
+
+4. File Analysis / Visualizations ― to present our users with their packet information and activity, we will put together a series of intuitive visualizations. These include:
+   * Geographical Map ― showing the country or region of packet source/destination
+   * Frequent Connections ― enumerating the most common connections made
+   * Website Security ― comparing protocols of websites visited by user
+
+
+### Team
 Authors: handeland@google.com, jevingu@google.com, mnuzen@google.com 
 Reviewers: arunkaly@, promanov@
 
+### Built With
+- [Maven](https://maven.apache.org)
+- [Java](https://java.com/en/download/faq/java8.xml)
+- [App Engine](https://cloud.google.com/appengine/docs/standard/java)
+
 ## Setup: 
```

Comment:
@mnuzen @thorncorona This section could use some work to make it more fluid. I remove any mention of the cloud bucket as that will no longer be available